pkg/agent/openflow/pipeline.go

```diff
@@ -968,12 +968,13 @@ func (c *client) bridgeAndUplinkFlows(uplinkOfport uint32, bridgeLocalPort uint3
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 		// Forward the packet to conntrackTable if it enters the OVS pipeline from the bridge interface and is sent to
-		// local Pods.
+		// local Pods. Set the packet with MAC rewrite mark, so that the dstMAC will be re-written with real MAC in
+		// the L3Routing table, and it could be forwarded to the valid OVS interface.
 		c.pipeline[ClassifierTable].BuildFlow(priorityHigh).
 			MatchProtocol(binding.ProtocolIP).
 			MatchInPort(bridgeLocalPort).
 			MatchDstIPNet(localSubnet).
-			Action().SetDstMAC(globalVirtualMAC).
```

Comment:
Hi @wenyingd . so what's the original dst MAC of the reply packet from kube-proxy?

Reply:
I forgot the result of packet capture. But I am sure it would not be the MAC of the destination Pod.